introduction/principles.md

```diff
@@ -33,27 +36,27 @@ it is, the more difficult it becomes for anyone to contribute.
 In order for guest teams to be able to contribute meaningfully to a project, the host team must be _transparent_.
 This means that guest teams should be able to have an understanding of:
 
-* The product and its direction
+* The project/repo and its direction
 * Outstanding feature requirements
 * Progress on feature requirements
 * Decision making of the host team
 
-When possible, the above should be communicated clearly and in details, from teams internal definitions of items to special case scenarios specific to the project, in a manner that can be easily understood to those that are not part of the core team.
+When possible, the above should be communicated clearly and in details, from teams internal definitions of items to special case scenarios specific to the project, in a manner that can be easily queried and understood to those that are not part of the core team.
 
 ## Prioritized Mentorship
 
 _Mentorship_ from host team to guest team via trusted committers is a key aspect of InnerSource.
-Contributors on guest teams are upleveled so that they understand enough about the host team system to change it successfully.
-In the process of doing so, they come to better understand the host team's software system as a general consumer.
-They can use it more effectively and help other consumers to do so as well.
+Contributors on guest teams are upleveled so that they understand enough about the host team's project/repo to change it successfully.
+In the process of doing so, they come to better understand the host team's software system as a general consumer and ambassador for the project/software.
+This individual committer can then over time and with experience take on a broader role in the project as a trusted committer.
```

Comment:
Think it's important to use the InnerSource naming throughout.

```suggestion
This individual contributor can then over time and with experience take on a broader role in the project as a trusted committer.
```

Reply:
You are soo correct. fixed now.

Reply:
Yup. If anyone has any additional suggestions please file them in a new PR.